app/style/video-calling.less

```diff
@@ -38,6 +38,16 @@
   border-radius: 4px;
 }
 
+.video-calling {
+  &--active {
+    position: absolute;
+    z-index: 101;
```

Comment:
Let's stick to our variables for this. We already noticed that the guest label is erroneously shown above the video calling ui.

Reply:
The `.full-screen` was also a good thing.